package.json

```diff
@@ -134,7 +134,7 @@
     "inquirer": "^8.0.0",
     "lerna": "^6.6.2",
     "lint-staged": "^12.1.7",
-    "listr2": "^5.0.3",
```

Comment:
I think we need to be careful here about respecting our `engines` requirements since `listr2@8` has a minimum Node.js version of 18 while Forge I think has a Node.js 16 requirement?

This might require a major version bump along the way.

Reply:
Node 16 is now [officially unsupported](https://nodejs.org/en/about/previous-releases). 🤔 I think it makes sense to bump our min supported version to Node 18 (separately) for changes like this one.

Reply:
It looks like the only thing in the listr2 v8 migration notes is the node requirement. I'm gonna see if pinning us on the most recent v7 minor version is a trivial change that could unblock this PR. If it looks hairy, I can hold off on this PR until we bump forge's min supported version

Reply:
Updated! 7.0.2 supports Node 16 and higher https://github.com/listr2/listr2/blob/listr2%407.0.2/package.json

Reply:
Awesome. Glad to see it was a small lift. Going to mark this as resolved then. 👍